client/src/shared/App.tsx

```diff
@@ -45,13 +46,14 @@ export default class App extends React.Component {
     return (
       <ThemeProvider theme={theme}>
         <Router>
-          <React.Fragment>
-            <Switch>
-              <Route exact path="/" component={Login}/>
-              <PrivateRoute component={TestSection} exact path="/test" isLoggedIn={true} />
-            </Switch>
-          </React.Fragment>
-        </Router>
+        <React.Fragment>
+          <Switch>
+            <Route exact path="/" component={Login}/>
+            <PrivateRoute component={TestSection} exact path="/test" isLoggedIn={true} />
+            <PrivateRoute component={VolunteerList} exact path="/volunteer-list" isLoggedIn={true} />
```

Comment:
can you change the route to `volunteers`

Reply:
got it!